src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/CustomAttributeBuilder.cs

```diff
@@ -14,14 +14,19 @@
 ===========================================================*/
 
 using System.Buffers.Binary;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using System.Text;
-using System.Diagnostics;
 
 namespace System.Reflection.Emit
 {
     public class CustomAttributeBuilder
     {
+        internal ConstructorInfo m_con;
+        internal object?[] m_constructorArgs;
```

Comment:
`m_constructorArgs` and `m_blob` can be private

Reply:
Changed.